include/spdlog/spdlog.h

```diff
@@ -46,6 +46,15 @@ inline std::shared_ptr<spdlog::logger> create(std::string logger_name, SinkArgs
 //   spdlog::initialize_logger(mylogger);
 SPDLOG_API void initialize_logger(std::shared_ptr<logger> logger);
 
```

Comment:
Should  be declared somewhere at the bottom

Reply:
From at the bottom, you mean at end of available methods?